hydra-node/src/Hydra/Network/Ouroboros.hs

```diff
@@ -254,9 +256,14 @@ withOuroborosNetwork tracer localHost remoteHosts networkCallback between = do
       , recvMsgDone = pure ()
       }
 
-data TraceOuroborosNetwork
+data WithHost trace = WithHost Host trace
+  deriving (Show)
+
+data TraceOuroborosNetwork msg
   = TraceSubscriptions (WithIPList (SubscriptionTrace SockAddr))
   | TraceErrorPolicy (WithAddr SockAddr ErrorPolicyTrace)
   | TraceAcceptPolicy AcceptConnectionsPolicyTrace
```

Comment:
Seeing the naming convention of ouroboros-network of likely being `XXXTrace` for trace types. Shall we adopt that as well? We have at least `TraceXXX` and `XXXLog` in use.

Reply:
No change required here, just starting the conversation :)